server/service/src/sync/api/get_site_info.rs

```diff
@@ -3,10 +3,21 @@ use serde::Deserialize;
 use super::*;
 
 #[derive(Debug, PartialEq, Deserialize)]
+#[serde(rename_all = "camelCase")]
 pub struct SiteInfoV5 {
     pub(crate) id: String,
-    #[serde(rename = "siteId")]
     pub(crate) site_id: i32,
+    pub(crate) initialisation_status: InitialisationStatus,
+}
+
+// SITE_INITIALISATION_STATUS.4dm
```

Comment:
:nit the comment could be more informative, it assumes that the reader knows what a 4dm file is and that it relates to mSupply. 
If it said `Taken from SITE_INITIALISATION_STATUS.4dm` or mentioned mSupply or said 'source of truth is..' maybe?

Reply:
Of course ! Thanks eab494ff6c2329dd416e1f2d93fe398cb6188aa8
Sometimes I forget that TMF won't be the only devs reading/working with this code